src/api/species/species.ts

```diff
@@ -9,45 +9,6 @@ import { GetUploadStatusResponsePayload, ResolveResponse, UploadFileResponse } f
  * surfaced to the caller via the requestSucceeded or error field.
  */
 
-const GET_SPECIES_ENDPOINT = '/api/v1/species';
```

Comment:
remove this file and directory?

Reply:
Yes, I didn't replace all the functions yet, that's why I didn't delete the file, but I will do it and remove it after that